pangeo_forge_recipes/recipes/xarray_zarr.py

```diff
@@ -666,9 +666,14 @@ def finalize_target(*, config: XarrayZarrRecipe) -> None:
         for dim in dims:
             arr = group[dim]
             attrs = dict(arr.attrs)
+            arr_copy = arr[:]
```

Comment:
```suggestion
            data = arr[:]
```

This is not actually a copy of the Zarr array but rather just a numpy array, so calling it a copy may be misleading.